src/templates/finops-hub/modules/hub.bicep

```diff
@@ -121,13 +128,16 @@ resource dataFactory 'Microsoft.DataFactory/factories@2018-06-01' = {
 module dataFactoryResources 'dataFactory.bicep' = {
   name: 'dataFactoryResources'
   params: {
-    dataFactoryName: dataFactoryName
+    dataFactoryName: dataFactory.name
     convertToParquet: convertToParquet
```

Comment:
If we're forcing parquet, shouldn't we remove this option? This can be a separate PR.